kategory-docs/docs/docs/typeclasses/functor/README.md

```diff
@@ -96,7 +96,7 @@ lifted(Option(1))
 
 #### Other combinators
 
-For a full list of other useful combinators available in `Functor` see the [`KDoc](/kdocs/typeclasses/functor)
+For a full list of other useful combinators available in `Functor` see the [KDoc](/kdocs/typeclasses/functor) 
```

Comment:
Have we KDocs online? or we should point to the source at GitHub?

Reply:
We don't have kdocs yet, pointing to the sources is fine IMO